DetectorDescription/DDCMS/src/MuonNumbering.cc

```diff
@@ -0,0 +1,48 @@
+#include "DetectorDescription/DDCMS/interface/MuonNumbering.h"
+#include "Geometry/MuonNumbering/interface/MuonBaseNumber.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include <iostream>
+
+using namespace cms;
+using namespace std;
+using namespace edm;
+
+const MuonBaseNumber
+MuonNumbering::geoHistoryToBaseNumber(const DDGeoHistory &history, MuonConstants& values) const {
```

Comment:
Can `MuonConstants` be changed to `const` if the code uses `find` instead of `operator[]`?